localstack/testing/snapshots/transformer_utility.py

```diff
@@ -105,6 +105,9 @@ def lambda_api():
         """
         return [
             TransformerUtility.key_value("FunctionName"),
+            TransformerUtility.key_value(
+                "CodeSize", value_replacement="<code-size>", reference_replacement=False
+            ),
```

Comment:
Had to introduce this because size of the packaged zip file varies a lot depending on the platform/installed zip binary and its defaults and it's not that important for us to compare against.